aep/general/0202/aep.md.j2

```diff
@@ -0,0 +1,112 @@
+# Fields
+
+The [`google.api.FieldInfo`][field info proto] type, through its accompanying
```

Comment:
Should we use google.api.FieldInfo or should we make aep.api.FieldInfo?

So far we've generally only kept google.* common components when they're privileged by protobuf, gRPC, or gRPC/JSON transcoding.  So i think the consistent thing to do would be to make an aep.api.FieldInfo.

Reply:
Alternatively, does it make sense to recommend protovalidate instead?  https://github.com/bufbuild/protovalidate/blob/main/proto/protovalidate/buf/validate/validate.proto#L2915

Reply:
I'd agree that we should move to `aep.api.FieldInfo` as opposed to keeping the `google.api.FieldInfo` message.

Here's some thoughts on protovalidate:
* There's a lot of great validations here.
* We don't own it just as we don't own google.api.FieldInfo. If there's ever a philosophical disagreement, we may have to tell people to change to `aep.api.FieldInfo` later on.
* There's *a lot* of validations here. We don't have guidance on following all of them and we'd have to keep a running list of which ones we agree with.

We might want to run this by the group, but I'd propose that we stick with `aep.api.FieldInfo`.

Reply:
I agree we don't own protovalidate (though @Alfus is part of our group and works at Buf).

protovalidate is also not really analogous to `FieldInfo`.  It's much, much more powerful.  If we get to the point where we can't see eye-to-eye with Buf on defining a well-known field format, and they don't offer some kind of plugin/extension mechanism, we could simply define CEL expressions in the AEPs.  Not as good an outcome, to be sure...but if we put aside organizational concerns and just focus on the technical details, protovalidate feels like a strictly and substantially better annotation than FieldInfo.

Reply:
After thinking about this more, `protovalidate` is more than fine. I've changed the language in the AEP to reflect this. (This also deleted most of the text in this AEP)